.rubocop.yml

```diff
@@ -1,21 +1,24 @@
 AllCops:
-  TargetRubyVersion: 2.3
-  TargetRailsVersion: 4.2
-  Include:
-    - '**/config.ru'
-    - '**/Rakefile'
-    - '**/*.rake'
   Exclude:
-    - 'db/**/*'
+    - 'db/migrate/*'
+    - 'db/schema.rb'
     - 'config/**/*'
     - 'script/**/*'
     - 'vendor/**/*'
+    - './.simplecov'
+    - 'client/node_modules/**/*'
+  TargetRailsVersion: 5.1
+  UseCache: true
+
+Bundler/OrderedGems:
+  Enabled: true
+  TreatCommentsAsGroupSeparators: false
 
 Style/StringLiterals:
   EnforcedStyle: double_quotes
 
 Style/FrozenStringLiteralComment:
-  Enabled: always
+  Enabled: false
```

Comment:
I'm curious as to why we're disabling this now.

Reply:
https://github.com/department-of-veterans-affairs/caseflow/blob/master/.rubocop.yml#L21

Reply:
What is the reasoning for having both repos use the same Rubocop settings? Frozen literals will be the default in Ruby 3.0, so it's generally recommended to have this setting turned on. If it used to be on before in this repo, why not leave it on? I currently have a work-in-progress PR to turn it on for the caseflow repo.

Reply:
Making frozen string literals default feature in Ruby 3 was an idea, but it was abonded due to 
compatibility issues. 
At Caseflow we strive to have all apps follow the same code styleguide/formatting rules. Hence the idea behind using 
the same rules for rubocop as well. But since we had it on and we'll planning on to change it in Caseflow repo, it makes sense
keeping it on. Thanks for pointing this out, Moncef. 
Also, please tag me on that PR of yours if you don't mind.